src/index.html

```diff
@@ -3,6 +3,12 @@
 <head>
   <!-- Run the app from the root directory -->
   <base href="/">
+
+  <script 
+    src="https://cdn.cookielaw.org/scripttemplates/otSDKStub.js" 
+    type="text/javascript" 
+    data-domain-script="94945f03-5e19-4775-ab08-18086adc0f18-test"
+  ></script>
```

Comment:
This should be removed as it will be injected by GTM shortly.

Reply:
Changes made in the recent commit `001d1f6`.